panaroo/integrate.py

```diff
@@ -0,0 +1,272 @@
+                          default=False) 
+
+    matching.add_argument(
+        "--length_outlier_support_proportion",
+        dest="length_outlier_support_proportion",
+        help=
+        ("proportion of genomes supporting a gene with a length more " +
+         "than 1.5x outside the interquatile range for genes in the same cluster"
+         +
+         " (default=0.01). Genes failing this test will be re-annotated at the "
+         + "shorter length"),
+        type=float,
+        default=0.1) 
+
+    parser.add_argument(
+        "--min_edge_support_sv",
+        dest="min_edge_support_sv",
+        help=(
+            "minimum edge support required to call structural variants" +
+            " in the presence/absence csv file (default=max(2, 0.01*n_samples))"
+        ),
+        default=2,
+        type=int) 
+
+    # MSA options
+    core = parser.add_argument_group('Gene alignment')
+    core.add_argument(
+        "-a",
+        "--alignment",
+        dest="aln",
+        help=("Output alignments of core genes or all genes. Options are" +
+              " 'core' and 'pan'. Default: 'None'"),
+        type=str,
+        choices=['core', 'pan'],
+        default=None) 
+    
+    core.add_argument(
+        "--aligner",
+        dest="alr",
+        help=
+        "Specify an aligner. Options:'prank', 'clustal', and default: 'mafft'",
+        type=str,
+        choices=['prank', 'clustal', 'mafft'],
+        default="mafft") 
+    
+    core.add_argument("--core_threshold",
+                      dest="core",
+                      help="Core-genome sample threshold (default=0.95)",
+                      type=float,
+                      default=0.95) 
+    
+    graph = parser.add_argument_group('Graph correction')
+    
+    graph.add_argument("--all_seq_in_graph",
+                       dest="all_seq_in_graph",
+                       help=("Retains all DNA sequence for each gene cluster in the graph " +
+                             "output. Off by default as it uses a large amount of space."),
+                       action='store_true',
+                       default=False)
+        
+    # Other options
+    parser.add_argument("-t",
+                        "--threads",
+                        dest="n_cpu",
+                        help="number of threads to use (default=1)",
+                        type=int,
+                        default=1) 
+    
+    parser.add_argument("--quiet",
+                        dest="quiet",
+                        help="suppress additional output",
+                        action='store_true',
+                        default=False)
+        
+    parser.add_argument("--version'",
+                        action="version",
+                        version='%(prog)s ' + __version__)
+
+    args = parser.parse_args()
+    return (args)
+
+
+def reformat_network(single_gml, output_dir, isolateName): #Generate network output needs to be reformatted for single gff inputs
```

Comment:
Add a short docstring to this function (see https://www.python.org/dev/peps/pep-0257/)